pages/signup.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Sign up | E-mart</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="../components.css" />
+    <link rel="stylesheet" type="text/css" href="https://style-ui.netlify.app/components.css">
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <div class="container-nav">
+        <span class="brandname">E-mart</span>
+        <div class="flex">
+            <input class="searchbar" type="text" placeholder="Search">
+            <i class="search-icon fas fa-search" aria-hidden="true"></i>
+        </div>
+        <div class="nav-options">
+            <a class="nav-item" href="./login.html"> Login </a>
+            <i class="nav-icon fa-regular fa-heart badge-icon">
+                <div class="badge-number">
+                    6
+                </div>
+            </i>
+            <i class="nav-icon fas fa-shopping-cart badge-icon" aria-hidden="true">
+                <div class="badge-number">
+                    1
+                </div>
+            </i>
+        </div>
+    </div>
+
+    <div class="page-container">
+
+        <div class="container-login">
+            <h2 class="heading">Sign up</h2>
+            <form>
+                <div>
+                    <label class="label">Email address</label>
+                    <br>
+                    <input class="text-input" type="email" placeholder="adarshbalika@gmail.com">
+                </div>
+                <div>
+                    <label class="label">Password</label>
+                    <br>
```

Comment:
avoid br tag and use margin or padding for that

Reply:
Noted. 👍